chia/wallet/wallet_node.py

```diff
@@ -86,6 +87,18 @@ def get_wallet_db_path(root_path: Path, config: Dict[str, Any], key_fingerprint:
     return path
 
 
+@streamable
+@dataclasses.dataclass(frozen=True)
+class Balance(Streamable):
+    confirmed_wallet_balance: uint128 = uint128(0)
+    unconfirmed_wallet_balance: uint128 = uint128(0)
+    spendable_balance: uint128 = uint128(0)
+    pending_change: uint64 = uint64(0)
+    max_send_amount: uint128 = uint128(0)
+    unspent_coin_count: uint32 = uint32(0)
+    pending_coin_removal_count: uint32 = uint32(0)
```

Comment:
previously we would also store `wallet_type` and `wallet_id` here. I suppose the `wallet_id` is now used as the dictionary index to get to this object, but what about wallet type, did we not use that for anything previously?

Reply:
Good catch, thanks! See [1cd08da](https://github.com/Chia-Network/chia-blockchain/pull/14631/commits/1cd08dab746621c6f6766fe7a6c97fae10cb9db3) i missed to add those two to the RPC call. So i fixed it and also added a test for the RPC call!